src/databases/ZipWrapper/avtZipWrapperFileFormatInterface.C

```diff
@@ -296,6 +306,14 @@ avtZipWrapperFileFormatInterface::Initialize(int procNum, int procCount,
     // Decide on root temporary directory
     if (tmpDir == "$TMPDIR" && procNum == 0)
     {
+#ifdef WIN32
```

Comment:
Do we really need conditional compilation to deal with `"TMP"` vs. `"TMPDIR"` vs. `"TEMP"`. Also, while here can you switch to use `VisItEnv::get()`?

Reply:
Thought it better than having if-tests I knew would fail on Windows, or non-Windows for that matter if I had the other platforms test for TMP or TEMP.

Reply:
Made the change to use Environment::get instead of getenv.